cloudconfig/templates.go

```diff
@@ -126,8 +126,8 @@ if [ -z "$CACHED_RUNNER" ];then
 	sudo ./bin/installdependencies.sh || fail "failed to install dependencies"
 else
 	sendStatus "using cached runner found in $CACHED_RUNNER"
-	sudo cp -a "$CACHED_RUNNER"  "/home/{{ .RunnerUsername }}/actions-runner"
-	sudo chown {{ .RunnerUsername }}:{{ .RunnerGroup }} -R "/home/{{ .RunnerUsername }}/actions-runner" || fail "failed to change owner"
+	ln -s "$CACHED_RUNNER"/actions-runner "/home/{{ .RunnerUsername}}/actions-runner" || fail "failed to create symlink"
```

Comment:
You can quote the entire path:

```bash
ln -s "$CACHED_RUNNER/actions-runner" "/home/{{ .RunnerUsername}}/actions-runner" || fail "failed to create symlink"
```

Variables cannot have slashes in them and the shell does the right thing.

Reply:
.. this change is very much relevant for us since it reduces our azure VM boot times quite significantly. However, we will need some more tweaks and changes, so I will close this one for now and create a new one once we're done.

Reply:
I wasn't suggesting to not add the change. It was just a nit which can be ignored. The change looked ok. Feel free to reopen when ready.